src/applications/ezr/tests/unit/config/helpers.spec.js

```diff
@@ -17,7 +18,8 @@ export const testNumberOfFields = (
 ) => {
   describe(`${pageTitle} page`, () => {
     it('should have appropriate number of fields', async () => {
-      const { container } = render(
+      const { container } = renderProviderWrappedComponent(
```

Comment:
I'm not sure why this change needed to be made, which essentially wraps the `DefinitionTestser` in a <`Provider>` with a mock store, but they were failing without it 🤷